fares-data-build-tool/create-data
# feat: Added back Button on operator settings & edit product details page

## Description

Added the back button for the operator settings and the pages navigated via the edit button from product details

## Testing instructions

Edit the settings in operator settings 
Click on edit button in product details page and the Back button will appear on the pages.
